### PR TITLE
Add CLI and redo application guide

### PR DESCRIPTION
**Pull request by thetechnick**
Signed-off-by: Nico Schieder <nschieder@redhat.com>

### CODE REVIEW

File: content/en/docs/concepts/package-format.md [hunk 1]
```diff
@@ -47,22 +47,6 @@ Well-Known Annotations are used to control the loading behavior of an object wit
 ```yaml
 apiVersion: manifests.package-operator.run/v1alpha1
 kind: PackageManifest
-catalog:
```
Comment (eqrx) [inline content/en/docs/concepts/package-format.md:50]: Pardon my ignorance, why is this removed?

File: content/en/docs/getting_started/api-reference.md [hunk 2]
```diff
@@ -225,6 +231,36 @@ status:
...
+kind: ClusterObjectSlice
+metadata:
+  name: example
+objects:
+- object:
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: example-deployment
+
```
Comment (eqrx) [inline content/en/docs/getting_started/api-reference.md:254]: Double newline is supposed to be here?

File: content/en/docs/guides/packaging-an-application.md [hunk 3]
```diff
@@ -4,73 +4,74 @@ weight: 100
...
+
+Packages may be cluster-scope, namespace-scope or both.  
+This controls whether you can install this package via `Package` or `ClusterPackage` API.  
+Namespaced packages can not contain cluster-scoped objects, like `Namespaces`.
+```yaml
 spec:
   scopes:
-  - Cluster
+  - Namespaced
+```
```
Comment (eqrx) [inline content/en/docs/guides/packaging-an-application.md:55]: Asked if the previous double newline is intentional because here is only one

**Review by eqrx — COMMENTED** [review]
Good documentation :) Only questions/nits

File: content/en/docs/concepts/package-format.md (hunk 1 above)
Comment (thetechnick) [reply content/en/docs/concepts/package-format.md:50]: Because it's not a thing ;)
